crates/sui-framework/tests/sui_system_tests.move

```diff
@@ -46,9 +46,24 @@ module sui::sui_system_tests {
 
         advance_epoch(scenario);
 
-        // After an epoch ends, report records are reset.
-        assert!(get_reporters_of(@0x2, scenario) == vector[], 0);
+        // After an epoch ends, report records are still present.
+        assert!(get_reporters_of(@0x2, scenario) == vector[@0x1], 0);
+
+        report_helper(@0x2, @0x1, false, scenario);
+        assert!(get_reporters_of(@0x1, scenario) == vector[@0x2], 0);
+
 
+        report_helper(@0x3, @0x2, false, scenario);
```

Comment:
super nit: could you add an assert here that the reporters of `@0x2` are `vector[@0x1, @0x3]`?